analysis/data-fetch.R

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Can we wrap this mechanism into a function that takes the time periods to be downloaded as an argument? I think it can live in the package and we can provide example usage in `README`.

Reply:
If I were just doing this analysis for myself, I would (ideally) use a makefile to automate/orchestrate the analysis. Is that a direction we want to consider for this project?

Reply:
I think GNU Make is a little too intimidating for newer users. That said, we do need to stay organized as we scale up. Maybe we can look at https://github.com/ropensci/drake?

Reply:
I added an argument to the function ```download_data``` which will allow users to specify the datasets to download. I will file an issue to use drake tomorrow.